.changeset/rude-beers-pull.md

```diff
@@ -0,0 +1,9 @@
+---
+'@livepeer/core-web': patch
```

Comment:
Ooh. What's this?

Reply:
Just the name of one of the core packages - renamed from `livepeer`